modules/osdk-common-prereqs.adoc

```diff
@@ -26,9 +26,9 @@ ifdef::golang[]
 endif::[]
 ifdef::ansible[]
 - link:https://docs.ansible.com/ansible/latest/index.html[Ansible] version v2.9.0+
-- link:https://ansible-runner.readthedocs.io/en/latest/install.html[Ansible Runner] version v1.1.0+
+- link:https://ansible-runner.readthedocs.io/en/latest/install.html[Ansible Runner] version v2.0.2+
```

Comment:
Can you confirm whether we're supposed to use `v` when specifying version number? I've been told to avoid this in the past, as per "Version, release, and modification identifiers" in the IBMSG. I understand in some instances that can't be avoided (mostly YAMLs), and with that  said you or (or past writers) have done this for cohesion. 

Any thoughts on it saying 'version v.'?

Reply:
Since this is how versions are used throughout the OSDK doc set, I am going to leave it for consistency. I did create a CIT ticket to confirm with engineering if we can remove the _v_ in the docs set after GA.